static/window/styles/core.css

```diff
@@ -2111,7 +2111,7 @@ body {
   padding: 0 1rem;
   align-items: center;
 }
-.setting__row__content--override-row__seperator{
+.setting__row__content--override-row__separator{
```

Comment:
Other side of the usage, CSS rules shouldn't be changed for compatability reasons.